README.md

```diff
@@ -67,6 +67,11 @@ oc create -f https://raw.githubusercontent.com/integr8ly/integreatly-operator/ma
 oc create -f https://raw.githubusercontent.com/integr8ly/integreatly-operator/master/deploy/role_binding.yaml
 ```
 
+Create a `Role` and `RoleBinding` for RHMDS
```

Comment:
Shouldn't this be created from a manifest?

Reply:
role is applied to the RHMCS namespace which doesn't do anything at the moment.

Reply:
but maybe it should ?